rosco-web/config/packer/aws-ebs.json

```diff
@@ -38,7 +38,8 @@
     "type": "shell",
     "script": "{{user `configDir`}}/install_packages.sh",
     "environment_vars": [
-      "deb_repo={{user `deb_repo`}}",
+      "repository={{user `repository`}}",
+      "package_type={{user `package_type`}}",
```

Comment:
Won't this fail unless you define `package_type` in the `variables` section up above (https://www.packer.io/docs/templates/user-variables.html)?